### DISCUSSION TITLE
Remove old hosts from MyFamily

### DISCUSSION THREAD

**Discussion by throny** [nusenu/ansible-relayor#235] [Ideas]
For better manageability of the MyFamily config it would be great to have a way to remove disabled relays with the playbook.

**Comment by nusenu** [comment]
you can easily remove obsolete fingerprints from your MyFamily
- for example after removing a server from your inventory -
by moving each instance's offlinemasterkeys folder (defaults to ~/.tor/offlinemasterkeys/<instance>
on the control machine) out of that folder

for example:
mkdir  ~/.tor/disabled/
mv ~/.tor/offlinemasterkeys/<instance> ~/.tor/disabled/

note: do NOT move the entire ~/.tor/offlinemasterkeys folder
and do NOT move the folder of an active tor instance, this would cause havoc

Does that address your goal?

**Comment by throny** [comment]
That's exactly what I wanted to achieve. Thanks a lot!
